src/sql-editor/types.ts

```diff
@@ -117,6 +117,18 @@ export enum OperatorType {
   Logical,
 }
 
+export enum MacroType {
+  Value,
+  Filter,
+  Group,
+}
+
+export enum ArgType {
+  Column = 'column',
+  RelativeTimeString = '5m',
+  Fill = 'previous',
```

Comment:
What's that Fill arg type?

Reply:
For the group macros you can specify how you want to fill missing values.

Looking at this now it's a bit weird having these arg types in an enum where the underlying value of the enum is the default. Don't know what I was thinking.